requirements/extra.txt

```diff
@@ -7,4 +7,4 @@ torchtext>=0.3.1, <0.7  # TODO: temporary fix fix for compatibility
 onnx>=1.7.0
 onnxruntime>=1.3.0
 hydra-core>=1.0
-https://github.com/facebookresearch/fairscale/archive/8e85ce8c93569017521d92ceb78dba2c57c955a0.zip # TODO temporary fix till release version
+https://github.com/tchaton/fairscale/archive/dev.zip
```

Comment:
what is the difference?
note that this will be preserved forever in Pypi, so if any time in the future you drop it installations fail!

Reply:
This is a good point, maybe it's right at this stage to create a FairScale fork with fixes in it within PL.

Reply:
do we have eta when FS will be out?